ns1/resource_zone.go

```diff
@@ -59,10 +59,6 @@ func resourceZone() *schema.Resource {
 				ForceNew: true,
 			},
 			// Computed
```

Comment:
Nitpick: We could also remove the `// Computed` comment here.

Reply:
👍